addons/mail/static/src/components/activity/activity.xml

```diff
@@ -128,14 +131,14 @@
                                 </Popover>
                             </t>
                             <t t-if="activityView.fileUploader">
-                                <button class="o_Activity_toolButton o_Activity_uploadButton btn btn-link" t-on-click="activityView.onClickUploadDocument">
+                                <button class="o_Activity_toolButton o_Activity_uploadButton btn btn-link btn-primary pt-0 pl-0" t-on-click="activityView.onClickUploadDocument">
                                     <i class="fa fa-upload"/> Upload Document
                                 </button>
                             </t>
-                            <button class="o_Activity_toolButton o_Activity_editButton btn btn-link" t-on-click="activityView.onClickEdit">
+                            <button class="o_Activity_toolButton o_Activity_editButton btn btn-link btn-primary pt-0" t-on-click="activityView.onClickEdit">
                                 <i class="fa fa-pencil"/> Edit
                             </button>
-                            <button class="o_Activity_toolButton o_Activity_cancelButton btn btn-link" t-on-click="activityView.onClickCancel" >
+                            <button class="o_Activity_toolButton o_Activity_cancelButton btn btn-link btn-primary pt-0" t-on-click="activityView.onClickCancel" >
```

Comment:
I really dislike changing padding on buttons because it changes the expected and natural click area.
I'll let this one pass if a redesign is planned but I still wanted to mention it.

Reply:
Also note that it would be nice if there was some margin between the buttons to avoid the clicking area to touch each other.